ee/get-support.md

```diff
@@ -35,11 +35,12 @@ To get the support dump from the Web UI:
 To get the support dump from the CLI, use SSH to log into a node and run:
 
 ```bash
+UCP_VERSION=$((docker container inspect ucp-proxy --format {% raw %}'{{index .Config.Labels "com.docker.ucp.version"}}'{% endraw %} 2>/dev/null || echo -n {{ page.ucp_version  }})|tr -d [[:space:]])
```

Comment:
@bkdocker: it works, but this got a little crazy.  do you still think this is a good idea?

Reply:
looks great